src/reusable-blocks/class-reusable-blocks-basket-handler.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace WPML\PB\Gutenberg;
+
+class Reusable_Blocks_Basket_Handler extends Reusable_Blocks_Handler {
```

Comment:
I don't like the use of the word "Handler".

Sometimes we can't avoid that, but whenever I can, I tend to follow [these advises](https://www.yegor256.com/2015/03/09/objects-end-with-er.html).

In this case, can't we call this class just `Reusable_Blocks_Basket`?

Also, didn't we agree to use cameCase for class and method names when they are under a namespace?

In fact, I think we should have a `\WPML\PB\Gutenberg\ReusableBlocks` namespace under which we have this class.

That would make easy to name this class something like `ManageBasket`, for instance. Or `HandleBasker`.  
Or even `BasketHandler` is we can't find a better name.

Reply:
Ok.

We can address that in the future then.

Reply:
At the time I started this project, we didn't agreed with the new format yet. I will add a commit to refactor all classes related to reusable blocks.

I renamed the classes and methods + extended the namespace => https://github.com/OnTheGoSystems/wpml-page-builders-gutenberg/pull/37/commits/39698cb5ac7e2e040004e3b84b46d321a4ca7c19